daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/ClosureConversionStackSafe.scala

```diff
@@ -0,0 +1,368 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.speedy
+
+/** Closure Conversion (Phase of the speedy compiler pipeline)
+  *
+  * This compilation phase transforms from SExpr0 to SExpr1.
+  */
+
+import com.daml.lf.data.Ref
+
+import com.daml.lf.speedy.SExpr.SCasePat
+import com.daml.lf.speedy.{SExpr0 => source}
+import com.daml.lf.speedy.{SExpr1 => target}
+
+import scala.annotation.tailrec
+
+private[speedy] object ClosureConversionStackSafe {
```

Comment:
The reason for the suffix was because during development I had two versions which I was comparing against at runtime.

At the end when I deleted the `Old` version, I thought keeping the suffix would help make the review diffs less confusing.

Anyway, I removed the suffix now.